edenia/antelope-tools
# 911 display information on geopmap

### What does this PR do?

- Resolve #911 

### Steps to test

1. Run locally, navigate to `http://localhost:3000/rewards-distribution`
2.  hover mouse over countries, daily rewards and paid producers and see popup
